gtest/src/manager.rs

```diff
@@ -610,6 +610,24 @@ impl ExtManager {
         *balance = balance.saturating_add(value);
     }
 
+    pub(crate) fn burn_from(&mut self, id: &ProgramId, value: Balance) {
+        let mut actors = self.actors.borrow_mut();
+        let (_, balance) = actors.get_mut(id).expect("Can't find existing program");
+
+        if *balance < value {
```

Comment:
I would instead use something like this here
```rust
balance.checked_sub(value).unwrap_or_else(...) // or expect()
```

Reply:
But why? Isn't `<` more straight forward?